### PR TITLE
Switch to minimal example data model

### PR DESCRIPTION
**Pull request by BrunoGrandePhD**
The data model is truly minimal and lacks niceties like descriptions and sources. I'm happy to add these in this PR, but I wanted to initiate a discussion on the attributes first. 

Example manifest templates generated from test dataset (syn24226514) using `--use_annotations`:

- [Patient](https://docs.google.com/spreadsheets/d/18WxvIiqb4NKUg5uCAC6D307NySmnKCGUwClYbCG4eD4/edit#gid=0)
- [Biospecimen](https://docs.google.com/spreadsheets/d/1CEY6eSVTQXkkAFJtbJngT58ZN-nbWj4cOp-iud0HYPY/edit#gid=0)
- [Bulk RNA-seq Assay](https://docs.google.com/spreadsheets/d/1ZApZjooPj8ADwSJLf417XCmh2vATbNhDh7Itg1RpZ4Y/edit#gid=0)

### CODE REVIEW

**Review by milen-sage — APPROVED** [review]

**Comment by BrunoGrandePhD** [comment]
I forget: was there a reason to hold back merging this PR? If not, I'll merge it and we can iterate on the minimal model later on.

**Comment by milen-sage** [comment]
Nope, no reason - it is approved and it can be merged.

**Comment by milen-sage** [comment]
As you said, we can iterate on features to the minimal schema later.

File: tests/test_utils.py [hunk 1]
```diff
@@ -330,3 +332,36 @@ def test_validate_property_schema(self, helpers):
...
+        This test also ensures that the CSV and JSON-LD
+        files for the example data model stay in sync.
+        """
+
+        # instantiate schema explorer
+        base_se = SchemaExplorer()
+
+        # load base schema (BioThings)
+        biothings_path = LOADER.filename("data_models/biothings.model.jsonld")
+        base_se.load_schema(biothings_path)
```
Comment (sujaypatil96) [inline tests/test_utils.py:351]: Would it be a good idea to refactor this code to use the [`get_schema_explorer()`](https://github.com/Sage-Bionetworks/schematic/blob/develop/tests/test_utils.py#L296) static method from the `Helpers` class?

**Review by sujaypatil96 — APPROVED** [review]
The comment I left isn't critical, but feel free to address it. I'm approving this PR to be merged.🚀

File: tests/test_utils.py (hunk 1 above)
Comment (BrunoGrandePhD) [reply tests/test_utils.py:351]: Yeah, absolutely! Do you mind committing that change and merging?

**Pull request merged**